lib/ts/MatcherUtils.cc

```diff
@@ -93,7 +93,8 @@ readIntoBuffer(const char *file_path, const char *module_name, int *read_size_pt
     // We don't want to signal this error on WIN32 because the sizes
```

Comment:
Should updated this comment and we don't support win32 anymore

Reply:
good point. updated